kml/serdes.cpp

```diff
@@ -674,15 +734,16 @@ void KmlParser::ParseAndAddPoints(MultiGeometry::LineT & line, std::string_view
     geometry::PointWithAltitude point;
     if (ParsePointWithAltitude(v, coordSeparator, point))
     {
-      // We don't expect vertical surfaces, so do not compare heights here.
-      // Will get a lot of duplicating points otherwise after import some user KMLs.
-      // https://github.com/organicmaps/organicmaps/issues/3895
-      if (line.empty() || !AlmostEqualAbs(line.back().GetPoint(), point.GetPoint(), kMwmPointAccuracy))
-        line.emplace_back(point);
+      line.emplace_back(point);
     }
```

Comment:
1. Is line.empty() check still needed?
2. Bracers are not needed anymore, and an else clause can be added with LOG(LWARNING, ("Can not parse KML coordinates from", v));

Reply:
> * Is line.empty() check still needed?

Here this code was removed...
But overall It is needed to fill the first point without checking the `!AlmostEqualAbs` that uses previous point.